reference/7/Microsoft.PowerShell.Utility/Get-Error.md

```diff
@@ -0,0 +1,144 @@
+---
+external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Utility
+ms.date: 11/12/2019
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/get-error?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-Error
+---
+
+# Get-Error
+
+## SYNOPSIS
+Gets and displays the most recent error messages from the current session.
+
+## SYNTAX
+
+```
+Get-Error [[-InputObject] <psobject>] [-Newest <Int>] [<CommonParameters>]
```

Comment:
This syntax does not match what I see in Preview.5. Is it changing the preview.6?